src/features/content-answers/ContentAnswersModalFooterActions.scss

```diff
@@ -0,0 +1,22 @@
+@import '../../styles/variables';
+
+.bdl-ContentAnswersModalFooterActions {
+    position: relative;
+    top: -$bdl-grid-unit * 4;
+    right: 0;
+    left: 0;
+    align-self: flex-start;
+    width: 100%;
+    padding: $bdl-grid-unit * 3 0 0 0;
+}
+
+.bdl-ContentAnswersModalFooterActions-button {
+    display: flex;
+    align-items: center;
```

Comment:
text should already be centered in the button right? I'm thinking these aren't needed